docs/release-notes.rst

```diff
@@ -1,3 +1,141 @@
+
+------
+
+**Error on EPG mapping delete operation**
+
+When a dynamic endpoint mapping is added to an application using the "Manage Endpoint Mappings", the application gets created on BIG-IP. If this mapping is deleted using the "RESET" button on "Manage Endpoint Mappings" form, user may encounter an error "The requested Pool Member (/Partition/App/Pool /NodePartition/NodeIP) was not found"
+
+**Workaround:** Click on the "Submit" button again, and the mapping will be reset properly without any errors. 
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/185
+ 
+------
+
+**AS3 application can either have static nodes or dynamic nodes but not both**
+
+AS3 applications will support either static nodes or dynamic nodes (Using Manage Endpoint Mappings button) but not both. 
+
+------
+
+**Dynamic endpoints will not be discovered if any of the dynamic endpoint IPs already belong to the static nodes on the BIG-IP**
+
+Dynamic endpoints are the endpoints present in APIC Endpoint Group. The app automatically updates this in the BIG-IP application’s pool members provided the correct association is configured via the application. But if this dynamic endpoint list consists of an IP which has already been added as a static node on the BIG-IP previously, none of the dynamic endpoints will get updated in the application. 
+
+**Workaround:** Ensure that the APIC endpoint subnet/IPs are different from the static endpoint IPs on the BIG-IP.
+
+------
+
+**Same Dynamic endpoint mappings on 2 separate partitions of a BIG-IP are not supported**
+
+For a single BIG-IP device, if 2 AS3 applications belonging to 2 different partitions are associated with the same APIC Endpoint Group (Tenant|Application|EPG), the dynamic discovery of nodes will not work for either of the AS3 applications. 
+
+**Workaround:** Remove erroneous applications and recreate with different mappings
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/187
+
+------
+
+**BIG-IP reboots OR BIG-IP services restart if more than 60 endpoints are dynamically discovered in an APIC EPG**
+
+If an APIC endpoint group has more than 60 endpoints attached, then the endpoint list will not get reflected on the BIG-IP, and users may experience service restarts on BIG-IP. 
+
+**Workaround:** For any dynamic endpoint mapping, please ensure that the number of endpoints in the corresponding APIC endpoint group never crosses 60.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/188
+
+------
+
+**Node not removed from BIG-IP pool when node IP is a substring of some other node's IP**
+
+If a node, for example node with IP 1.2.3.4 is deleted from APIC, and there is also another node 1.2.3.40 of which the original IP is a substring, it may be possible that dynamic end point attach detach feature is not able to delete 1.2.3.4 from BIG-IP. Note: The pool members will get deleted as expected. 
```

Comment:
If a node, for example a node with an IP of 1.2.3.4, is deleted...

..it may be possible that the dynamic endpoint attach feature...